README.md

```diff
@@ -257,6 +257,49 @@ Settings:
 
 Clone repo or pull latest changes to specified repo
 
+#### git.blame-copy
+
+Command for creating copies of a file with git blame history saving.
+
+Original script written in bash:
+https://dev.to/deckstar/how-to-git-copy-copying-files-while-keeping-git-history-1c9j
+
+Usage:
+```shell
+  inv git.blame-copy <path to original file> <path to copy>,<path to copy>...
+```
+
+Settings:
+
+* `copy_commit_template` template for commits created during command workflow
+* `copy_init_message_template` template for init message printed at command start
+
+Template variables:
+
+* `action` - The copy algorithm consists of several intermediate actions
+(creating temporary files, merging commits, etc.)
+The `action` variable stores the header of the intermediate action.
+* `original_path` - Contains value of first argument of the command
+(path of original file that will be copied)
+* `destination_paths` - Sequence of paths to which the original file will be copied
+* `jira_task` - jira task that will be parsed from current git branch.
+If no task found in branch, then will be empty
```

Comment:
Jira task is not the only one project management site, let's find more generic name